swagger.yaml

```diff
@@ -3671,6 +3671,8 @@ definitions:
         x-go-custom-tag: gorm:"type:text"
         type: string
         description: Array of image statuses.
+      kube_key_namespace:
```

Comment:
It's not user facing info it should be in https://github.com/openshift/assisted-service/blob/master/internal/common/db.go#L65-L68

Reply:
That's done to reduce the DB calls as discussed in: https://github.com/openshift/assisted-service/pull/1941#discussion_r646541878
@tsorya @filanov should I return the additonal 'common.GetHostFromDB' db call then?

Reply:
Sorry missed the other comment, i wrote a reply.

Reply:
ok, so moved to common.Host and used common.GetHostFromDB again then.